checkov/terraform/checks/resource/aws/AppLoadBalancerTLS12.py

```diff
@@ -0,0 +1,33 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class AppLoadBalancerTLS12(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that load balancer is using TLS 1.2"
```

Comment:
btw, why not 1.3? 1.2 had the heartbleed vulnerability as far as i remmember

Reply:
i was going off their list https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html. I double checked on the UI there's no 1.3 option

Reply:
pff. I hoped AWS handled that. 
Thanks for verifying this one. merging :)